src/core/types/DataTypes.sol

```diff
@@ -9,7 +9,6 @@ import { UD60x18 } from "@prb/math/src/UD60x18.sol";
 //
 // - Lockup
 // - LockupDynamic
-// - LockupLinear
```

Comment:
This should not be removed. `LockupLinear` library still exists in this contract.

Reply:
https://github.com/sablier-labs/v2-core/pull/1079/commits/24c3b89a405c49cfe737314b7fdf7faa77eecd8d

Reply:
mistake, ty